lib/studies/reply/mail_view_page.dart

```diff
@@ -18,25 +18,27 @@ class MailViewPage extends StatelessWidget {
     return Scaffold(
       body: SafeArea(
         bottom: false,
-        child: Container(
-          color: Theme.of(context).cardColor,
-          child: Padding(
-            padding: const EdgeInsetsDirectional.only(
-              top: 42,
-              start: 20,
-              end: 20,
-            ),
-            child: Column(
-              crossAxisAlignment: CrossAxisAlignment.start,
-              children: [
-                _MailViewHeader(
-                  email: email,
-                ),
-                const SizedBox(height: 32),
-                Expanded(
-                  child: _MailViewBody(message: email.message),
+        child: SizedBox(
```

Comment:
This feels a little hacky. I think the better API would be to use `SingleChildScrollView.constraint` or  `IntrinsicHeight`

Reply:
Discussed this in https://github.com/Renzo-Olivares/gallery/pull/22